### PR TITLE
picom: new command page

### PR DESCRIPTION
**Pull request by ErlanRG**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/picom.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Standalone compositor for Xorg.
+> More information: <https://wiki.archlinux.org/title/picom>.
+
+- Enable picom during a session:
+
+`picom &`
+
+- Start picom as a background process:
+
+`picom &`
```
Comment (mfrw) [inline pages/linux/picom.md:12]: I think the above example and this one is exactly same, can we probably club them together :)

File: pages/linux/picom.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Use custom configuration file:
```
Comment (mfrw) [inline pages/linux/picom.md:14]: Maybe:
```suggestion
- Use a custom configuration file:
```

File: pages/linux/picom.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# picom
```
Comment (mfrw) [inline pages/linux/picom.md:1]: Thank you for your contribution. A few cosmetic nits though :)

File: pages/linux/picom.md (hunk 1 above)
Comment (ErlanRG) [reply pages/linux/picom.md]: My bad, sorry! These should be two different commands
`picom &` and `picom -b`. For some reason I kept them the same. Fixing that right away

**Review by ErlanRG — COMMENTED** [review]
32b5beb corrects the issues.

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks

File: pages/linux/picom.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+> Standalone compositor for Xorg.
+> More information: <https://wiki.archlinux.org/title/picom>.
+
+- Enable picom during a session:
```
Comment (kbdharun) [inline pages/linux/picom.md:6]:
```suggestion
- Enable `picom` during a session:
```

File: pages/linux/picom.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+`picom &`
+
+- Start picom as a background process:
```
Comment (kbdharun) [inline pages/linux/picom.md:10]:
```suggestion
- Start `picom` as a background process:
```

File: pages/linux/picom.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# picom
+
+> Standalone compositor for Xorg.
+> More information: <https://wiki.archlinux.org/title/picom>.
```
Comment (kbdharun) [inline pages/linux/picom.md:4]: I generally would have suggested linking to the [project](https://github.com/yshui/picom), but Arch wiki seems to have better documentation here, so IG this link is fine.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution, I have some minor suggestions for this page.

**Pull request merged**